data/prefixes.js

```diff
@@ -659,12 +659,18 @@ f(prefixPointer, browsers =>
 // Text decoration
 let prefixDecoration = require('caniuse-lite/data/features/text-decoration')
 
-f(prefixDecoration, browsers =>
+f(prefixDecoration, { match: /x.*#[235]/ }, browsers =>
+  prefix(['text-decoration-skip', 'text-decoration-skip-ink'], {
+    feature: 'text-decoration',
+    browsers
+  })
+)
+
+let prefixDecorationShorthand = require('caniuse-lite/data/features/mdn-text-decoration-shorthand')
```

Comment:
Now uses data from MDN via `caniuse-lite`:
- `text-decoration` shorthand
- `text-decoration-color`
- `text-decoration-line`
- `text-decoration-style`

My intention was to change only the data source and nothing else.